### PR TITLE
feat: set up the project with create-react-app

### LINKED ISSUE #5: Set up the project using `create-react-app`
We want to setup the project with `create-react-app`.

### PR DESCRIPTION
**Pull request by henites72**
Closes #5

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (Buranch) [inline public/robots.txt]: @henites72 I'm not sure about this file. Do we really need it?

Comment (Buranch) [inline public/robots.txt]: @henites72 I'm not sure if we need this file. Let me know if it's necessary.

File: src/serviceWorker.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```
Comment (Buranch) [inline src/serviceWorker.js]: We can also remove this file.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```
Comment (Buranch) [inline src/index.js]: Shoud be removed, if you delete serviceWorker.js

File: src/logo.svg [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (Buranch) [inline src/logo.svg]: Feel free to remove this file as well.

File: public/manifest.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (Buranch) [inline public/manifest.json]: We can replace it with our project title.

File: public/robots.txt (hunk 1 above)
Comment (henites72) [reply public/robots.txt]: Yeah, we don't need it.

File: src/logo.svg (hunk 4 above)
Comment (henites72) [reply src/logo.svg]: The logo is displayed in the app, so lets not remove it.

File: src/serviceWorker.js (hunk 2 above)
Comment (henites72) [reply src/serviceWorker.js]: Yup

**Pull request merged**